experimental/DoubleAndHyperComplexes/test/linear_strands.jl

```diff
@@ -0,0 +1,42 @@
+@testset "linear strands of complexes of graded modules" begin
```

Comment:
@wdecker : First version of linear strands are now here. I'm working on the induced maps.